### DISCUSSION TITLE
Fillter Not Worked

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6118] [Help]
![11](https://user-images.githubusercontent.com/29665710/143422217-f7dc0770-8db1-4154-aab0-923af0621e63.jpg)
This My Code
![12](https://user-images.githubusercontent.com/29665710/143478390-4ef2f090-cdbe-4776-93b6-7c6942bf1887.jpg)
![13](https://user-images.githubusercontent.com/29665710/143422785-97d8fc1f-dbb1-4b71-974c-3901815b00c8.jpg)

**Comment by VictorTomaili** [comment]
Try to use constructor injection with `IHttpContextAccessor` interface instead creating new `HttpContextAccessor`

**Reply by ra-bedo11** [reply]
> Try to use constructor injection with `IHttpContextAccessor` interface instead creating new `HttpContextAccessor`

Why 

This Ok
But now 
The problem in this filter  i want distincit
![11](https://user-images.githubusercontent.com/29665710/143478218-2927f192-7dd6-4d04-bc4b-6983d1a066c0.jpg)
